src/components/Card/index.tsx

```diff
@@ -62,29 +69,36 @@ function Card({
   }
   return (
```

Comment:
I suspect you can split this component up to make it more compact 🤔

Reply:
I tried and it looks a bit better but I guess it can be refactored and simpler..